lib/ops/opsservice/operationgroup.go

```diff
@@ -63,10 +64,18 @@ func (s swap) Check() error {
 
 // createSiteOperation creates the provided operation if the checks allow it to be created
 func (g *operationGroup) createSiteOperation(operation ops.SiteOperation) (*ops.SiteOperationKey, error) {
+	return g.createSiteOperationWithOptions(operation, createOperationOptions{})
+}
+
+type createOperationOptions struct {
+	force bool
```

Comment:
So you don't want to move it to `operationGroup`? It could be an aspect that cuts not only through the operation creation but perhaps something else - if you look at it that way.

Reply:
Not really, I'm not sure it belongs as an operation group property, because operation group is maintained per operation, to me it's more like a property of the "create" call which is why I opted to make it this way.